app/src/main/java/fr/neamar/kiss/dataprovider/ContactsProvider.java

```diff
@@ -87,7 +84,7 @@ public void requestResults(String query, Searcher searcher) {
                 if (fuzzyScore.match(pojo.phoneSimplified, matchInfo)) {
                     match = true;
                     pojo.relevance = matchInfo.score;
-                    pojo.setDisplayNameHighlightRegion(0, 0);
+                    pojo.setNameHighlight(0, 0);
```

Comment:
Isn't it better to use `clearNameHighlight()` instead?

Reply:
Yes, absolutely. Forgot I created that function.